ultimate_cron.routing.yml

```diff
@@ -21,3 +21,14 @@ entity.ultimate_cron_job.edit_form:
     _title: 'Edit job'
   requirements:
     _entity_access: 'ultimate_cron_job.edit'
+
+entity.ultimate_cron_job.run:
+  path: '/admin/config/system/cron/jobs/manage/{ultimate_cron_job}/run'
+  defaults:
+    _controller: '\Drupal\ultimate_cron\Controller\JobController::runCronJob'
+  options:
```

Comment:
This is only needed when it doesn't match the name, this should work automatically.